static/config/status-chain-genesis.json

```diff
@@ -7,25 +7,10 @@
       "balance": "0x1B1AE4D6E2EF500000"
     }
   },
-  "coinbase": "0x0000000000000000000000000000000000000000",
   "config": {
-    "byzantiumBlock": 1700000,
-    "chainId": 777,
-    "daoForkSupport": true,
-    "eip150Block": 0,
-    "eip150Hash": "0x41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d",
-    "eip155Block": 10,
-    "eip158Block": 10,
-    "ethash": {},
-    "homesteadBlock": 0
```

Comment:
Why have you deleted all that?

Reply:
You [asked me](https://github.com/status-im/status-go/pull/397#discussion_r144716617)  to make genesis as [simple as possible](https://github.com/status-im/status-go/pull/397#discussion_r144716639), so I left only the minimal required options.
All the rest will be set to their default values. Default value for the stated above options is 0.

Reply:
No, I haven't. I've asked it to be as close to public testnets as possible and proposed some specific values to test here: https://github.com/status-im/status-go/pull/397#discussion_r144716639

The reason is that `nil` is the default, not `0`: https://github.com/status-im/status-go/blob/90acfedf7ac60ab641e9db4cac84d7db1bad6d8e/vendor/github.com/ethereum/go-ethereum/params/config.go#L107 and `nil` would be incorrect in our case.

Reply:
The very first revision of genesis was fully based on Ropsten genesis used in the project. I only changed chain id, pre-allocated accounts and extra data :) The options you proposed significantly differ from the default values of Ropsten.

Would be so kind as to specify exactly which values would you like me to use?

If I get it correctly, all you really want at the moment is to add the options with their corresponding values as follows:

"config": {
    "byzantiumBlock": 1700000,
    "chainId": 777,
    "daoForkSupport": true,
    "eip150Block": 0,
    "eip150Hash": "0x41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d",
    "eip155Block": 10,
    "eip158Block": 10,
    "ethash": {},
    "homesteadBlock": 0
  },

Reply:
The reason why I've requested that change is because Ropsten has already reached forks specified in its genesis, for example `byzantiumBlock`, `eip150Block`, `eip155Block` and `eip158Block` and its actual head is about 1882811. 
While our private net with the same config won't reach those forks unless we specify 0 block there. So we need to modify our genesis so that those forks have already been reached from the very beginning.

Summarily, although my changes significantly differ from the ones from Ropsten, they allow to reach almost the same state in which Ropsten is in at the moment.

Makes sense?

Reply:
Got it. Just in case, would it be ok if I set the values as follows:
"config": {
"chainId": 777
"homesteadBlock": 0
"daoForkBlock": 0
"daoForkSupport": true
"byzantiumBlock": 0
"eip150Hash": "0x41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d",
"eip155Block": 0
"eip158Block": 0
}

Reply:
I pushed all the fixes discussed. Could you please prove me right or wrong on eip150Hash. In order to set its value I did the following:
1. Ran the StatusChain network 
2. Attached to it
3. called eth.getBlock(0) and retrieved the hash => "0x28c4da1cca48d0107ea5ea29a40ac15fca86899c52d02309fa12ea39b86d219c"

Reply:
Sounds good, let's see :)